pkg/stack_mgr/stack.go

```diff
@@ -226,3 +228,33 @@ func (s *Stack) PrintOutputs(ctx context.Context) {
 		logrus.Printf("%s: %s", k, v)
 	}
 }
+
+func (s *Stack) Print(ctx context.Context, name string, tablePrinter *util.TablePrinter) error {
+	stackOutput, err := s.GetOutputs(ctx)
+
+	if err != nil {
+		return err
+	}
+	url := stackOutput["frontend_url"]
+	status := s.GetStatus()
+	meta, err := s.Meta(ctx)
+	if err != nil {
+		return err
+	}
+	tag := meta.DataMap["imagetag"]
+	imageTags, ok := meta.DataMap["imagetags"]
+	if ok && len(imageTags) > 0 {
+		var imageTagMap map[string]interface{}
+		err = json.Unmarshal([]byte(imageTags), &imageTagMap)
+		if err != nil {
+			return err
```

Comment:
errors.wrap this one since it crosses the boundary between external code to our code